README.md

```diff
@@ -222,10 +222,29 @@ fn plugin_main(context: PluginContext) -> Result<(), Box<dyn Error>> {
 }
 ```
 
+> **IMPORTANT:** Compiled REAPER Plug-in Extensions (IE, `.dll` files) must be prefixed with `reaper_` in order for REAPER to load them during startup. Naming the library `reaper_my_extension` in `Cargo.toml` will result in the compiled file being named `reaper_my_extension`, thus obeying this rule. If you rename your library, make sure that the compiled file placed in `REAPER/UserPlugins` is prefixed with `reaper_` before attempting to test it.
+
 The macro primarily exposes an `extern "C" ReaperPluginEntry()` function which calls
 `reaper_low::bootstrap_extension_plugin()`. So if for some reason you don't want to use that
 macro, have a look at the macro implementation. No magic there.
 
+#### Step-by-Step Instructions
+
+The following instructions should result in a functional extension, loaded into REAPER on start:
+
+1. Make a new directory for the project.
```

Comment:
Steps 1 to 3 can be simplified by making use of `cargo new reaper-my-extension --lib`.